### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by yyyaohhh**
#122

### CODE REVIEW

File: docs/diagrams/ModelClassDiagram.png
Comment (rocketninja7) [inline docs/diagrams/ModelClassDiagram.png:1]: Might want to make it more obvious which side the multiplicity is for.

File: docs/ModelClassDiagram/Model.png
Comment (rocketninja7) [inline docs/ModelClassDiagram/Model.png:1]: Looks the same as the other image?

File: docs/diagrams/ModelClassDiagram.png
Comment (yyyaohhh) [reply docs/diagrams/ModelClassDiagram.png:1]: Updated👍

File: docs/ModelClassDiagram/Model.png
Comment (yyyaohhh) [reply docs/ModelClassDiagram/Model.png:1]: Resolved

**Review by xCOLOURx — APPROVED** [review]
LGTM

**Pull request merged**